apps/dashboard/app/views/batch_connect/session_contexts/_favorites.html.erb

```diff
@@ -0,0 +1,5 @@
+<span role='button'
+  class='clickable text-wrap' 
+  data-api-url=<%= path.path %>>
```

Comment:
I think instead of catching this in javascript we can use the rails route to get the correct url -  i.e., `files_path(path.to_s ,fs: path.filesystem)`

Reply:
This is part of my confusion with your redefinition comment - this doesn't work for the manually/redefined paths because I'm not sure how to provide them with the filesystem attribute (hence the JavaScript override).

Reply:
>  I'm not sure how to provide them with the filesystem attribute (hence the JavaScript override).

They default to `fs` so users defining them (like we do at OSC) have that default.

https://github.com/OSC/ondemand/blob/d291b215eaa24b76743e45e45f85167437ec13cb/apps/dashboard/app/apps/favorite_path.rb#L10

Then remote FavoritePaths filesystems initialize with the right filesystem.

https://github.com/OSC/ondemand/blob/d291b215eaa24b76743e45e45f85167437ec13cb/apps/dashboard/config/initializers/rclone.rb#L20